### DISCUSSION TITLE
一直返回账号密码错误。。。

### DISCUSSION THREAD

**Discussion by Lunamio** [lxysl/Auto_Attendance#23] [Q&A]
作者你好，我很确定我的账号和密码是对的但它一直返回错误，请问在设置secrets的时候应该在哪一栏呢？actions，codespaces还是dependabot？

**Comment by lxysl** [comment]
![Snipaste_2021-03-26_16-50-07](https://user-images.githubusercontent.com/45818075/112606436-5ab25b00-8e53-11eb-9162-5784ca5ecbd0.jpg)
应该是在secrets->actions里
